tasks/kernel_matrix_testing/platforms.py

```diff
@@ -26,7 +26,15 @@ def get_platforms():
         return cast("Platforms", json.load(f))
 
 
-def filter_by_ci_component(platforms: Platforms, component: Component) -> Platforms:
+class KMTTestJob:
+    def __init__(self, name, arch, test_set, kernels):
```

Comment:
Could we have type annotations for these values? Specially for the arch one, to avoid mistakes later.